src/flag_gems/runtime/register.py

```diff
@@ -1,37 +1,39 @@
-import torch
-
 from . import backend, commom_utils, error
 from .backend.device import DeviceDetector
 
-aten_lib = torch.library.Library("aten", "IMPL")
-
 
 class Register:
     def __init__(
         self,
         config,
         user_unused_ops_list=None,
         lib=None,
-        debug=False,
+        debug=True,
     ):
         # lib is a instance of torch.library.Library
-        self.lib = lib
         self.device = DeviceDetector()
+        self.vendor_list = list(commom_utils.vendors_map)
+        self.check_backend()
+        self.lib = lib
         # reg_key like 'CUDA', reg_bac_key like AutogradCUDA
         self.reg_key = self.device.name.upper()
         self.reg_bac_key = commom_utils.autograd_str + self.reg_key
-        self.vendor_list = list(commom_utils.vendors_map)
         self.debug = debug
         self.forward_ops = []
         self.backward_ops = []
-        self.config = config
         self.vendor_extend_configs = self.get_vendor_extend_op()
         self.vendor_unused_ops_list = self.get_vendor_unused_op()
         self.unused_ops = user_unused_ops_list + self.vendor_unused_ops_list
-        self.check_backend()
-        self.for_each(config)
+        self.config = config + self.vendor_extend_configs
```

Comment:
do we allow vendors to have their own operators?

Reply:
yes，This is for 
1. if the implementation of large sections of its code is different from that of nv
2. An operator with good performance for a vendor but low performance on nv

Reply:
I got it. it actually means the specific implementation by vendors, instead of extension. I suggest rename 'op's and 'config's carefully.